src/main/java/com/discordbolt/api/command/CommandListener.java

```diff
@@ -25,22 +25,23 @@ public void onMesageEvent(MessageReceivedEvent e) {
             return;
         }
 
-        // Message is just a single prefix.
-        if (message.length() <= 1) {
+        // Message is shorter than or equal to prefix.
+        if (message.length() <= manager.getCommandPrefix(e.getGuild()).length()) {
             return;
         }
 
         // Check if message started with our command prefix
-        if (message.charAt(0) != manager.getCommandPrefix(e.getGuild())) {
+        if(!message.startsWith(manager.getCommandPrefix(e.getGuild()))) {
```

Comment:
This method is case sensitive, and while I know it was also case sensitive before, having any length prefix would make having letters in the prefix more common. IMO we should make this case insensitive by converting the message and prefix to lowercase.